public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js

```diff
@@ -201,7 +179,7 @@ export default class AlertsDashboardFlyoutComponent extends Component {
     const queryParamsString = queryString.stringify(params);
     history.replace({ ...this.props.location, search: queryParamsString });
 
-    httpClient.get('../api/alerting/alerts', { query: params }).then((resp) => {
+    httpClient.get('../api/alerting/alerts', { query: params })?.then((resp) => {
```

Comment:
Should we be concerned if the result if this get call is undefined?

Reply:
This was actually a means of resolving the unit test failure. It seems that newer versions of node don't support uncaught promises or errors in tests. Before making this change, the test was failing with the following error
https://github.com/opensearch-project/alerting-dashboards-plugin/actions/runs/5281727443/jobs/9555698123?pr=576#step:11:414
```
/home/runner/work/alerting-dashboards-plugin/alerting-dashboards-plugin/OpenSearch-Dashboards/plugins/alerting-dashboards-plugin/public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js:4367
              }).then(function (resp) {
                ^

TypeError: Cannot read properties of undefined (reading 'then')